app/scrapers/sachsen_scraper.rb

```diff
@@ -0,0 +1,187 @@
+    people = [meta_text.partition(/\p{Upper}{2}/)[0].strip]
+    parties = [meta_text.match(/\p{Upper}\p{Lower}*\p{Upper}+\s*\p{Upper}*/)[0].strip]
+    originators = { people: people, parties: parties}
+  end
+
+  def self.extract_overview_paper(item, doctype)
+    meta_text = extract_meta_text item
+    meta_text = meta_text[6..meta_text.length].strip
+    full_reference = extract_reference(meta_text)
+    {
+      legislative_term: full_reference.split('/')[0],
+      full_reference: full_reference,
+      reference: full_reference.split('/')[1],
+      doctype: doctype,
+      title: extract_title(item),
+      url: nil,
+      published_at: extract_date(meta_text),
+      originators: extract_originators(meta_text),
+      is_answer: nil,
+    }
+  end
+
+  def self.extract_detail_paper(item)
+    meta_text = extract_meta_text(item)
+    type = extractType(meta_text)
+    extract_overview_paper(item, type)
+  end
+
+  def self.switchToTerm(i, m, top)
+    nav = top.frame_with(name: 'head').click
+    nav = m.get("/redirect.aspx?wahlperiode=" + i.to_s)
+    nav.forms.first.submit
+  end
+
+  class Detail < DetailScraper
+    def scrape
+      m = mechanize
+      top = m.get BASE_URL
+      SachsenScraper.switchToTerm(@legislative_term, m, top)
+      content = top.frame_with(name: 'content').click
+      content = searchDetail(content)
+      paper = nil
+      SachsenScraper.extract_overview_items(content).each do |item|
+        paper = SachsenScraper.extract_detail_paper(item)
+        content = m.click(item.search('//a').first)
+        answeredDate = content.search('//*[@id="ctl00_masterContentCallback_content_tabTreffer_trefferDataView_IT0_HyperLink6"]').first
+        if (answeredDate.nil? || answeredDate.text.empty?)
+          paper[:is_answer] = false
+        else
+          paper[:is_answer] = true
+          paper[:published_at] = SachsenScraper.extract_date(answeredDate.text)
+          top = m.get(extract_viewer_url(content))
+          pdfurl = extract_pdf_url(top)
+          paper[:url] = pdfurl
+        end
+      end
+      paper
+    end
+
+    def extract_pdf_url(top)
+      nav = top.frame_with(name: 'navigation').click
+      onloadValue = nav.search("//body").first.attribute("onload")
+      pdfurl = onloadValue.to_s[/(http\S*?\.pdf)/]
+    end
+
+    def extract_viewer_url(content)
+      pdfTable = content.search('//table[@id="ctl00_masterContentCallback_content_tabTreffer_trefferDataView_IT0_anzeige_tblButtons"]').first
+      viewerIds = pdfTable.search('//input').select do |i|
+        !i.attribute('name').nil? && !i.attribute('name').value.nil? && !i.attribute('name').value.index(/no\$btn/).nil?
+      end
+      viewerIds = viewerIds.map do |i|
+        i.attribute('name').value
+      end
+      viewerId = viewerIds.last.match(/anzeige\$(.*?)_(.*?)_Drs_(.*?)_no\$btn/)
+      "http://edas.landtag.sachsen.de/viewer.aspx?dok_nr=" + viewerId[1] + "&dok_art=Drs&leg_per=" + viewerId[3] + "&pos_dok="+ viewerId[2]
```

Comment:
Surrounding space missing for operator '+'.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.